### PR TITLE
[SPARK-35950][WebUI] Failed to toggle Exec Loss Reason in the executors page

### PR DESCRIPTION
**Pull request by pingsutw**
### What changes were proposed in this pull request?

Update the executor's page, so it can successfully hide the "Exec Loss Reason" column.

### Why are the changes needed?

When unselected the checkbox "Exec Loss Reason" on the executor page,
the "Active tasks" column disappears instead of the "Exec Loss Reason" column.  

Before:
Screenshot from 2021-06-30 15-55-05
After:
Screenshot from 2021-06-30 22-21-38

### Does this PR introduce _any_ user-facing change?

Yes, The Web UI is updated.

### How was this patch tested?

Pass the CIs.

### CODE REVIEW

File: core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html [hunk 1]
```diff
@@ -128,7 +128,7 @@ <h4 class="title-table">Executors</h4>
               Shuffle Write</span></th>
           <th>Logs</th>
           <th>Thread Dump</th>
-          <th>Exec Loss Reason</th>
+          <th><span data-toggle="tooltip" data-placement="top"></span>Exec Loss Reason</th>
```
Comment (gengliangwang) [inline core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html:131]: Why do we need this?

**Comment by gengliangwang** [comment]
Thanks for the fix.
cc @holdenk @dongjoon-hyun @BryanCutler

**Comment by holdenk** [comment]
Jenkins ok to test

File: core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html (hunk 1 above)
Comment (pingsutw) [reply core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html]: Reverted it.

**Review by gengliangwang — APPROVED** [review]
LGTM. Thanks for the fix.

**Comment by gengliangwang** [comment]
Merging to master

**Comment by dongjoon-hyun** [comment]
Oh, sorry for missing your ping, @gengliangwang .

**Comment by dongjoon-hyun** [comment]
Thank you for the fix, @pingsutw .

**Pull request closed without merging**